better-code/src/chapter-2-contracts.md

```diff
@@ -967,9 +973,9 @@ understood, is another source of complexity. In fact we should
 probably put a link in the documentation to a definition.
 
 ```swift
-/// - Precondition: `areInIncreasingOrder` is [a strict weak
-///   ordering](https://simple.wikipedia.org/wiki/Strict_weak_ordering)
-///   over the elements of `self`.
+  /// - Precondition: `areInIncreasingOrder` is [a strict weak
```

Comment:
What's the rationale for these spaces?  I don't see how they're helpful.  Personally I think they look quite awkward in the rendered output.

Reply:
OK, it's consistent with the rest of the chapter (except for the several(!) code blocks where indentation is totally messed up).  But maybe we should reconsider and strip the leading indentation from code blocks.  Horizontal space is at a premium in print books (do we still believe in those?) and anyway to my eye, isolated indentation looks weird on a gray backgound and does nothing useful on a white background.